tiny_dnn/activations/selu_layer.h

```diff
@@ -0,0 +1,74 @@
+/*
+    Copyright (c) 2013, Taiga Nomi and the respective contributors
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
+#pragma once
+#include "tiny_dnn/activations/activation_layer.h"
+#include "tiny_dnn/layers/layer.h"
+
+namespace tiny_dnn{
+
+	class selu_layer : public activation_layer{
+	public:
+		
+		selu_layer(const float_t lambda=1.05070,const float_t alpha=1.67326)
```

Comment:
@srk97 please add documentation. Follow this as a reference:
https://github.com/tiny-dnn/tiny-dnn/blob/master/tiny_dnn/activations/leaky_relu_layer.h